terraso_backend/apps/core/models/__init__.py

```diff
@@ -8,6 +9,7 @@
     "Landscape",
     "LandscapeGroup",
     "Membership",
+    "SlugModel",
```

Comment:
Question: Shy is SlugModel here, does this means there a table for that in the DB? or why do we need it here?